docs/docset.yml

```
@@ -6,6 +6,7 @@ toc:
   - toc: reference
   - toc: release-notes
 subs:
+  version: "9.0.0"
```

IMO we will likely need to be more specific, per discussions in https://github.com/elastic/docs-builder/issues/737
```suggestion
  version-90: "9.0.0"
```

---

This was intended to be a temporary solution until https://github.com/elastic/docs-builder/issues/737 is addressed and there's a central place to manage versions. 

I had to add `version` subs to multiple repos in [related PRs](https://github.com/search?q=%22%5Bdocs%5D+Fix+various+syntax+and+rendering+errors%22+user%3Aelastic+is%3Apull-request&type=issues). I'm not planning to go back and update all other repos unless https://github.com/elastic/docs-builder/issues/737 can't be addressed before the next release, but you're welcome to make this change in this repo (and any other repos) if you think that would be better.